conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/utils/actions.py

```diff
@@ -1,8 +1,9 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
All our files should be starting with the license header. Can you add here on in a followup PR?